rad/prelude/io.rad

```diff
@@ -1,25 +1,29 @@
 {:module 'prelude/io
  :doc "Some basic I/O functions."
- :exports '[print! shell! process! read-line! send-code! read-code!
+ :exports '[print! shell! process! read-line! read-file-value! read-file-values! send-code!
             shell-with-stdout! shell-no-stdin! process-with-stdout! write-file!
            ]}
 
 (import prelude/patterns :unqualified)
 
-;; read-line
 (def read-line!
   "Read a single line of input and interpret it as radicle data."
-  (fn [] (read (get-line!))))
+  (fn [] (read-annotated "read-line!" (get-line!))))
 
-(def read-code!
-  "Read code (as data) from a file. Returns a vector of expressions"
-  (fn [filename]
-    (read-many (read-file! filename))))
+(def read-file-value!
+  "Read a single radicle value from a file."
+  (fn [file]
+    (read-annotated file (read-file! file))))
```

Comment:
This will wrap the filename in square brackets which is in contrast to `load!`. Is this what we want?